.github/workflows/integration-tests.yml

```diff
@@ -244,6 +244,7 @@ jobs:
         env:
           TRITON_INTERPRET: "1"
         run: |
+          TRITON_INTERPRET=1 pytest -vvv python/tutorials/06-fused-attention.py::test_op
           cd python/test/unit
```

Comment:
I think you can just run it with other interpreter tests below to enjoy the parallelism?

Reply:
good point, let me try that